TESTS/events/queue/main.cpp

```diff
@@ -31,54 +31,60 @@ using namespace utest::v1;
 volatile bool touched = false;
 
 // static functions
-void func5(int a0, int a1, int a2, int a3, int a4) { 
+void func5(int a0, int a1, int a2, int a3, int a4)
+{
     touched = true;
     TEST_ASSERT_EQUAL(a0 | a1 | a2 | a3 | a4, 0x1f);
 }
 
-void func4(int a0, int a1, int a2, int a3) {
+void func4(int a0, int a1, int a2, int a3)
+{
     touched = true;
-    TEST_ASSERT_EQUAL(a0 | a1 | a2 | a3, 0xf); 
+    TEST_ASSERT_EQUAL(a0 | a1 | a2 | a3, 0xf);
 }
 
-void func3(int a0, int a1, int a2) {
+void func3(int a0, int a1, int a2)
+{
     touched = true;
     TEST_ASSERT_EQUAL(a0 | a1 | a2, 0x7);
 }
 
-void func2(int a0, int a1) {
+void func2(int a0, int a1)
+{
     touched = true;
     TEST_ASSERT_EQUAL(a0 | a1, 0x3);
 }
 
-void func1(int a0) {
+void func1(int a0)
+{
```

Comment:
I was curious about this so I tried astyle with both brace styles:
```
with style=kr:     1841070 line changes
with style=attach: 1819522 line changes
```

It would actually take fewer code changes to change the existing KR braces than to standardize on KR. We should reconsider our stance on brace placement.

I've never understood why our brace style is inconsistent between functions and every other syntactic construct.

Reply:
> It would actually take fewer code changes to change the existing KR braces than to standardize on KR. We should reconsider our stance on brace placement.

I have testing online compiler - we need to match what we have here and what online compiler does. It seems it defines a rule to actually attach `{` in the header files, will run few more tests and get its options that it defines

To have to have both aligned.

Reply:
I updated this PR, attached C++ as the online compiler does.

Reply:
This contradict the guidelines and guidelines are made to be followed; the purpose of this PR is to enforce them. If the online compiler is wrong; fix the online compiler. Not the other way around.

Reply:
Updated, KR style braces are still the minority, although only slightly:
```
with style=kr:     1819759  line changes
with style=attach: 1819522 line changes
```

I'm suggesting we change the guidelines to match the style of code users are contributing. I had no idea C++ inlines are a separate style, and if we don't know what our rules are on braces how are our users supposed to?

I mean what is this:

``` cpp
namespace hi {

class Thing {
    int dothing() {
        if (thing) {
            blahblahblah;
        } else {
            blahblahblah;
        }
    }
};

int dothing()
{
    while (thing) {
        blahblahblah;
    }
}

}
```
What's the reason for not making squiggly brackets consistent?